docs/community/tsc-membership.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Joining the ByConity Technical Steering Committee
+tags:
+  - Docs
+---
+
+# Joining the ByConity Technical Steering Committee
+
+This guide aims to provide information on how to become a member of the ByConity Technical Steering Committee (TSC). 
+It outlines the development roles in ByConity, the definition of the TSC, why it was formed, its basic principles, 
+benefits of being a TSC member, and the steps to become a TSC member.
+
+## Development Roles in ByConity
+* **Users:** Individuals who use ByConity's software and provide feedback in the form of bug reports and feature suggestions. 
+* **Contributors:** Members of the community who contribute code, documentation, pull request reviews, designs, 
+or other technical artifacts to the ByConity initiative.
+* **Committers:** Contributors who have earned the ability to modify ("commit") source code, documentation, 
+or other technical artifacts in a project's repository.
+* **Maintainers:** Committers who have been elected due to their merit and contributions towards the development and evolution of the project. 
+They have write access to the code repository, can vote on community-related decisions, and can propose other active contributors for committership.
+
+## What is the Technical Steering Committee?
+The TSC is a group of maintainers who contribute to the maintenance and governance of the ByConity Initiative. 
+It is responsible for making decisions that affect the entire initiative, rather than just a single project. 
+TSC members commit to participating in the voting process.
+
+## Why the TSC was formed
+The TSC was formed when ByConity was OpenSourced in 2023 to ensure a system of open governance that promotes ownership 
+and neutrality of all participants in the initiative.
+
+There are some basic principles of the TSC members
+* No founding members with special privileges
+* Equality among all contributors
+* Asynchronous decision making 
+* Every maintainer is allowed to automatically become a TSC member
+
+## Benefits of becoming a TSC member
+
+As a TSC member you will enjoy the following benefits: 
+ 
+* Recognition from ByConity community
+* Participation in the voting process on topics concerning the entire initiative
+* Included in key decision-making for the Initiative. 
+
+## How to become a TSC member
+
+Any maintainer can become a TSC member. To become a TSC member, you need to become a maintainer of one of the projects in the ByConity Initiative.
+
```

Comment:
I'm a little confused about what roles can be called as TSC member? Only maintainer or other roles(i.e. contributor, commitor) also are TSC member?

Reply:
No, a TSC member must be a maintainer, but a maintainer may not be a TCS member.